content/en/docs/helm/helm_init.md

```diff
@@ -0,0 +1,9 @@
+---
+section: deprecated
+---
+
+## helm init
+
+This command does not exist in Helm 3, following the [removal of Tiller](https://helm.sh/docs/faq/#removal-of-tiller). You no longer need to install Tiller in the cluster and set up a Helm state before in order to use Helm.
```

Comment:
`set up a Helm state before in order to use Helm`

Does this mean...

"set up an initial Helm state in order to use Helm"

"set up a Helm state before using Helm"

or something else?

Reply:
should probably read "You no longer need to install Tiller in your cluster in order to use Helm."

Reply:
@bridgetkromhout I assume the former... I copied the words of @hickeyma from [here](https://github.com/helm/helm-www/issues/726#issuecomment-648141171). If the extra context is confusing, I could shorten it to:

> You no longer need to install Tiller in the cluster ~~and set up a Helm state before~~ in order to use Helm.